docs/continuous-delivery/gitops/secret-management/secret-management-gitops.md

```diff
@@ -0,0 +1,177 @@
+# Secret Management with GitOps
+
+GitOps primarily have git repositories in the center considered a source of truth for managing both infrastructure and application. This Infrastructure and application will require secured access to other resources of the system through secrets. Committing clear text secrets into git repositories is unacceptable even if the repositories are private to your team and organization. Teams need a secure way to handle secrets when using GitOps.
```

Comment:
```suggestion
GitOps primarily have git repositories in the center considered a source of truth for managing both infrastructure and application. This infrastructure and application will require secured access to other resources of the system through secrets. Committing clear-text secrets into git repositories is unacceptable even if the repositories are private to your team and organization. Teams need a secure way to handle secrets when using GitOps.
```

Reply:
@John-Garland a tip when you review... if you use suggestions like above, the author can click on commit suggestion to easily incorporate your comments... (it's the (+-) button when you add a comment)